client/src/app.jsx

```diff
@@ -42,27 +45,41 @@ class App extends React.Component {
         this.setState({
           productId: results[3].data.id,
           productInformation: results[3].data,
-          styles: results[4].data
+          styles: results[4].data,
+          qNa: this.props.qNaTestData
         });
-        // console.log(results);
       })
       .catch((err) => {
         console.log('this is the err 🥲 ', err)
       });
   }
 
+  searchQuestionHandler(newState) {
+    console.log(newState)
+    this.setState({
+      qNa: newState
+    })
+
+  }
+
   render() {
     return (
       <div className='app'>
         <Overview state = {this.state}/>
         <RelatedProducts state={this.state} />
-        <QuestionsNAnswers data={this.state.qNa}/>
+        <QuestionsNAnswers data={this.state.qNa} searchQuestionHandler={this.searchQuestionHandler}/>
         <RatingsAndReviews />
       </div>
     )
   }
 }
 
+App.propTypes ={
+  qNaTestData: propTypes.array.isRequired
+}
+
+
+
 
 ReactDOM.render(<App qNaTestData={qNa}/>, document.getElementById('app'));
 
```

Comment:
Do you still need test data passed down here? Or if it is live data being passed down, maybe rename?